index.ts

```diff
@@ -5,7 +5,7 @@ import dotenv from 'dotenv';
 dotenv.config()
 
 const botToken = process.env.DISCORD_TOKEN
-const client = new discordClient({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES] });
+const client = new discordClient({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES, Intents.FLAGS.DIRECT_MESSAGES], partials: ['CHANNEL', 'MESSAGE', 'USER'] });
```

Comment:
What to the partials do? Not familiar with this. I assume you added this so you could listen to the user's dm channel?

Reply:
No idea TBH. I went looking to find out why my DMChannel was never sending the bot any messages and someone else was asking the same question. The response to fix it was to add partials and the intents.

Reply:
I have removed the partials and it appears as though the messages still come through, so let's try without them for now.